src/server/tablet_snap.rs

```diff
@@ -260,7 +260,7 @@ async fn recv_snap_files(
                 None => return Err(box_err!("missing chunk")),
             };
             f.write_all(&chunk[..])?;
-            limit.consume(chunk.len());
+            limit.consume(chunk.len()).await;
```

Comment:
Should we add consume in line 254 as well?

Reply:
I think it isn't necessary, the first chunk has little message.